jwtauth/jwtauth.go

```diff
@@ -0,0 +1,56 @@
+// Package jwtauth implements an interface for providing CFSSL
+// authentication.
+package jwtauth
+
+import (
+	//"flag"
```

Comment:
Should remove commented out imports (or just run `goimports` on every save to automatically add/remove imports).